goethe/src/test/java/com/palantir/goethe/GoetheBootstrapTest.java

```diff
@@ -87,8 +87,29 @@ public void testFormatting(FormatterFacade formatter) {
                         + "class Foo {\n"
                         + "    static {\n"
                         + "        System.out.println(\n"
-                        + "                \"" + padding + "\");\n"
+                        + "                \"" + longWord + "\");\n"
                         + "    }\n"
                         + "}\n");
     }
+
+    @ParameterizedTest
+    @MethodSource("formatterFacades")
+    public void testFormatJavadoc(FormatterFacade formatter) {
```

Comment:
Any reason to not simply add this to the existing test?

Reply:
It feels better to have a separate test for a separate piece of functionality to test. The test name and targeted logic can clarify what's being tested.
And if there are more things that we want to verify in the future, it's better to have a precedent of separate tests than a big code block that contains all the needed constructs. Though granted this probably won't happen.
The downside is just some extra test code duplication.